evennia/commands/cmdsethandler.py

```diff
@@ -351,6 +351,14 @@ def update(self, init_mode=False):
                     elif path:
                         cmdset = self._import_cmdset(path)
                         if cmdset:
+                            if cmdset.key == '_CMDSET_ERROR':
+                                # If a cmdset fails to load, check if we have a fallback path to use
+                                fallback_path = settings.CMDSET_FALLBACKS.get(path, None)
```

Comment:
@TehomCD Using _IN_GAME_ERRORS is a good idea. I was already working on integrating this PR when I read that, so I took the opportunity to add that as well when I was at it, thanks!